desktop/libs/notebook/src/notebook/connectors/sql_alchemy_tests.py

```diff
@@ -367,7 +367,7 @@ def test_empty_database_names(self):
       'options': {'url': 'phoenix://'}
     }
 
-    snippet = Mock()
+    snippet = MagicMock()
```

Comment:
This is the new change please approve it also @romainr . Found this link https://stackoverflow.com/questions/17181687/mock-vs-magicmock useful for this.

Reply:
Yes, and good description!

The issue was that snippet is a dictionary object, sometimes we can mock it like below but dict are simpler to mock by dicts.

```
snippet = Mock(
  type='phoenix'
)
```